app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new(session[:user_params])
```

Comment:
Line 3: is the session coming from the create action?

Reply:
Yes